components/SocialMedia.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-card-actions>
+    <v-container>
+      <v-icon
+        class="mx-5"
+        tag="a"
+        href="https://www.facebook.com/recepcaoimeusp"
+        target="_blank"
+        color="#fff"
+        size="35px"
+        >fa-brands fa-facebook</v-icon
+      >
+
+      <v-icon
+        class="mx-5"
+        tag="a"
+        href="https://www.instagram.com/imeusp.recepcao/"
+        target="_blank"
+        color="#fff"
+        size="35px"
+        >fa-brands fa-instagram</v-icon
+      >
+      <v-icon
+        class="mx-5"
+        tag="a"
+        href="https://discord.gg/qKXAkf8EjE"
+        target="_blank"
+        color="#fff"
+        size="35px"
+        >fa-brands fa-discord</v-icon
+      >
+      <v-icon
+        class="mx-5"
+        tag="a"
+        href="https://www.tiktok.com/@imeusp.recepcao"
+        target="_blank"
+        color="#fff"
+        size="35px"
+        >fa-brands fa-tiktok</v-icon
+      >
+    </v-container>
+  </v-card-actions>
+</template>
+
+<script>
+export default {
+  name: 'SocialMedia',
```

Comment:
Como o padrão de cada ícone se repete, podemos adicionar as informações que mudam (link e nome do ícone) nos dados, para, posteriormente, renderizar cada componente com base nesses dados.

```suggestion
  name: 'SocialMedia',
  data: () => ({
    icons: [
      {
        name: 'fa-brands fa-facebook',
        link: 'https://www.facebook.com/recepcaoimeusp',
      },
      {
        name: 'fa-brands fa-instagram',
        link: 'https://www.instagram.com/imeusp.recepcao/',
      },
      {
        name: 'fa-brands fa-discord',
        link: 'https://discord.com/invite/qKXAkf8EjE',
      },
      {
        name: 'fa-brands fa-tiktok',
        link: 'https://www.tiktok.com/@imeusp.recepcao',
      },
    ],
  }),
```